src/c++/perf_analyzer/genai-perf/tests/test_llm_metrics.py

```diff
@@ -273,46 +273,46 @@ def test_openai_llm_profile_data(self, mock_read_write: pytest.MonkeyPatch) -> N
         assert metrics.output_token_throughputs_per_request == pytest.approx(ottpr)
         ott = [9 / ns_to_sec(14)]
         assert metrics.output_token_throughputs == pytest.approx(ott)
-        assert metrics.num_output_tokens == [3, 6]
-        assert metrics.num_input_tokens == [3, 4]
+        assert metrics.output_sequence_lengths == [3, 6]
+        assert metrics.input_sequence_lengths == [3, 4]
 
         assert stat["time_to_first_token"]["avg"] == pytest.approx(4.5)  # type: ignore
         assert stat["inter_token_latency"]["avg"] == pytest.approx(3)  # type: ignore
         assert stat["output_token_throughput_per_request"]["avg"] == pytest.approx(  # type: ignore
```

Comment:
Should we also rename things like this to be output sequence throughput etc?

Reply:
I think that the throughput metric here makes sense to contextualize in tokens. 
Its the number of tokens thru the system per request.
Num_output_tokens is counting a length, so the migration to output_sequence_length makes sense to me imo.

Reply:
@matthewkotila There is new name for this metric called "TPS/user" and will be handled by a separate ticket (TMA-1951).